### DISCUSSION TITLE
Host Chaining

### DISCUSSION THREAD

**Discussion by nicwhites** [isontheline/pro.webssh.net#906] [Ideas]
What would be nice is a good gui element to link hosts together. I work behind multiple hosts and it would be awesome to go through them quicker with a simple button click. The idea is similar to the host changing feature in terminus.

For reference: 
https://support.termius.com/hc/en-us/articles/4407561837209-Host-Chaining

**Comment by isontheline** [comment]
Hello @nicwhites 👋

Thank you so much for your feedback 🙏

I've opened several months ago an issue about it : https://github.com/isontheline/pro.webssh.net/issues/577

I need to find some time to implement it
